src/zampy/datasets/ecmwf_dataset.py

```diff
@@ -120,11 +120,19 @@ def load(
                 files += (ingest_dir / self.name).glob(f"{self.name}_{var}*.nc")
 
         ds = xr.open_mfdataset(files, chunks={"latitude": 200, "longitude": 200})
-        ds = ds.sel(time=slice(time_bounds.start, time_bounds.end))
 
+        # rename valid_time to time
+        if "valid_time" in ds.dims:
+            ds = ds.rename({"valid_time": "time"})
+
+        ds = ds.sel(time=slice(time_bounds.start, time_bounds.end))
         grid = xarray_regrid.create_regridding_dataset(
             make_grid(spatial_bounds, resolution)
         )
+
+        # this is needed before regrid
+        ds = ds.unify_chunks()
```

Comment:
Didn't realize this would matter. What kind of error did you get before?

Reply:
The function `xrray_regird.utils.format_for_regrid` gets for chunksizes as `orig_chunksizes = obj.chunksizes`, see [here](https://github.com/xarray-contrib/xarray-regrid/blob/1425e1353b4405b4ea734ef37f78cf4538a72949/src/xarray_regrid/utils.py#L252C5-L252C37). If there are different chunk sizes, this will return an error.